test/parallel/test-https-truncate.js

```diff
@@ -67,6 +67,6 @@ const test = common.mustCall(function(res) {
   res.on('data', function(chunk) {
     bytes += chunk.length;
     this.pause();
-    setTimeout(this.resume.bind(this), 1);
+    setTimeout(() => this.resume(), 1);
```

Comment:
Nit: Can you please wrap `this.resume()`  in `{` and `}` since there's no need for the function to return a value?